playbooks/generic-build_images.yml

```diff
@@ -11,10 +15,29 @@
         - always
         - imagebuilder
         - upstream_patching
-  roles:
-    - role: suse-build-images
-      tags:
-        - imagebuilder
+
+    # The certificate generated before also needs to be known by the osh-deployer
+    - name: If developer mode, configure osh-deployer to include certs
+      include_role:
+        name: setup-caasp-workers
+        tasks_from: copy-certificates
+      when:
+        - developer_mode
+
+    - name: If developer mode, install docker
+      include_role:
+        name: install-docker
+        apply:
+          tags:
+            - docker
+      when:
+        - developer_mode
+
+    - name: If developer mode, build suse images given in extravars or suse-build-images defaults.
+      include_role:
+        name: suse-build-images
+        apply:
+          tags:
+            - imagebuilder
```

Comment:
Where's the check for `developer_mode` ?

Reply:
ditto.